addon/utils/element/estimate-element-height.js

```diff
@@ -1,8 +1,8 @@
-import { assert } from 'smoke-and-mirrors/-debug/helpers';
+import Ember from 'ember';
 
 export default function estimateElementHeight(element, fallbackHeight) {
-  assert(`You called estimateElement height without a fallbackHeight`, fallbackHeight);
-  assert(`You called estimateElementHeight without an element`, element);
```

Comment:
Actually - those imports should get removed, will have a look at why that's not happening